### DISCUSSION TITLE
Проблема

### DISCUSSION THREAD

**Discussion by kovalserg** [vvip-68/LedPanelWiFi#22] [Общие вопросы]
Здравствуйте! помогите деревянному, или дайте путь решения проблемы с компиляцией!
`Arduino: 1.8.13 (Windows 10), Плата:"NodeMCU 1.0 (ESP-12E Module), 80 MHz, Flash, Disabled (new aborts on oom), Disabled, All SSL ciphers (most compatible), 32KB cache + 32KB IRAM (balanced), Use pgm_read macros for IRAM/PROGMEM, 4MB (FS:2MB OTA:~1019KB), 2, v2 Lower Memory, Disabled, None, Only Sketch, 115200"

In file included from sketch\LedPanelWiFi_v1.14.ino.cpp:1:

C:\Users\User\AppData\Local\Arduino15\packages\esp8266\hardware\esp8266\3.1.2\cores\esp8266/Arduino.h:27:10: fatal error: stdlib.h: No such file or directory

   27 | #include <stdlib.h>

      |          ^~~~~~~~~~

compilation terminated.

exit status 1

Ошибка компиляции для платы NodeMCU 1.0 (ESP-12E Module).

Этот отчёт будет иметь больше информации с
включенной опцией Файл -> Настройки ->
"Показать подробный вывод во время компиляции"
Спасибо большое! второй вечер бьюсь безрезультатно.
`

**Comment by vvip-68** [comment]
Могу предложить попробовать поставить Arduino IDE 1.8.19

**Reply by kovalserg** [reply]
Спасибо за совет! Вечером попробую, но у меня стоит какая то из старых версий, просит постоянно обновить.

**Reply by kovalserg** [reply]
Спасибо еще раз! на другом пк поставил версию  Arduino IDE 1.8.19 и скетч скомпилировался! с ошибками, но все работает! спасибо большое еще раз!!!!!

**Comment by godaleks** [comment]
Добрый день. Установил прошивку на контроллер. Но больше недели мучался и не смог подключиться с телефона не через PaneAP не через wifi домашний. По монитору порта все отображалось и работало. Причина оказалась в телефоне. У жены такой-же телефон но он без проблем подключился к точке доступа контроллера. 
Может кто сможет помочь хотя бы в направлении куда копать? Что может в телефоне блокировать программу. Пробовал разные программы от гайвера и для лампы но они не работают!

**Comment by CoolerrAVR** [comment]
Добрый день!! Проблема. В монитор порта ничего не выводится, стартовая инфа с ESP8266 выводится, а дальше тишина. Установлены две IDE 2.3.7 и 1.8.19 обе молчат. Так вся система работает.

**Reply by vvip-68** [reply]
В настройках скетча отключен вывод отладочной информации в монитор порта?
Где-то стоит
#define DEBUG_SERIAL 0
?

**Reply by CoolerrAVR** [reply]
> В настройках скетча отключен вывод отладочной информации в монитор порта? Где-то стоит #define DEBUG_SERIAL 0 ?

Спасибо!! Всё завелось, хотя стояла до этого 1, залил 0107 потом сразу 1210.

**Comment by CoolerrAVR** [comment]
Новая беда, Файлы на флешке находит, но пишет ошибка формата и почему-то не видит расширение файла.
![Jinx!](https://github.com/user-attachments/assets/322134dd-ecfe-4669-b97b-c80f58c96802)

**Reply by vvip-68** [reply]
Ну, значит структура файла не соответствует ожидаемой структуре кадра размером 20x37
Или файл не является картинкой сохраненной из Jinx!, либо сохраняли не в том формате.
Утилитой JinxFramer файл открывается? Отображается правильно?

**Comment by CoolerrAVR** [comment]
Спасибо!!! Я как на картинках делал 2304, а нужно было оставить как прога сама циферки выставляет.
